docs/migrating/migrating_from_ez_publish_platform.md

```diff
@@ -225,11 +263,26 @@ Add the following new bundle to your new kernel file, `<new-ez-root>/app/AppKern
 
 ##### 3.1. Execute update SQL
 
-Import to your database the changes provided in one of the following files. It's also recommended to read inline comments as you might not need to run some of the queries:
+Import to your database the changes provided in one of the following files. It's also recommended reading inline comments as you might not need to run some queries *(if you for instance upgrade from 5.4.11)*:
+
+Postgres:
+- `<new-ez-root>/vendor/ezsystems/ezpublish-kernel/data/update/postgres/dbupdate-5.4.0-to-6.13.0.sql`
+- `<new-ez-root>/vendor/ezsystems/ezpublish-kernel/data/update/postgres/dbupdate-6.13.0-to-7.5.7.sql`
+
+MySQL/MariaDB:
+- `<new-ez-root>/vendor/ezsystems/ezpublish-kernel/data/update/mysql/dbupdate-5.4.0-to-6.13.0.sql`
+- `<new-ez-root>/vendor/ezsystems/ezpublish-kernel/data/update/mysql/dbupdate-6.13.0-to-7.5.7.sql`
+
 
-`MySQL: <new-ez-root>/vendor/ezsystems/ezpublish-kernel/data/update/mysql/dbupdate-5.4.0-to-6.13.0.sql`
+!!! note  "Change to UTF8mb4 for MySQL/MariaDB"
```

Comment:
This is not the only thing needed.

I think we should just instruct user to run all upgrade things mentioned at https://doc.ezplatform.com/en/latest/updating/4_update_database/, but skip [LandingPage -> Page builder upgrade](https://doc.ezplatform.com/en/latest/updating/4_update_2.2/#page-builder) mentioned for 2.2

Then continue here afterwards

Reply:
So I'm trying to provide a fast pass here as many steps there are not relevant when coming directly from 5.4.

We might be missing instructions for creating form container thought: https://doc.ezplatform.com/en/latest/updating/4_update_2.3/#form-builder

Can add that in `3.2.3 Add other eZ Enterprise schemas (eZ Enterprise only)`

Reply:
But why should you skip all of that?
You for sure need to upgrade db schema to 2.5 ( or no migration scripts will run at all). But  you should  for instance also create new top folder for "Forms" etc. and everything else explained there in order to get a fully functional 2.5

Reply:
But why should you skip all of that?
You for sure need to upgrade db schema to 2.5 ( or no migration scripts will run at all). But  you should  for instance also create new section for "Forms" (2.4) etc. and everything else explained there in order to get a fully functional 2.5.

Reply:
> You for sure need to upgrade db schema to 2.5 ( or no migration scripts will run at all)

There is:
```
MySQL/MariaDB:
- `<new-ez-root>/vendor/ezsystems/ezpublish-kernel/data/update/mysql/dbupdate-5.4.0-to-6.13.0.sql`
- `<new-ez-root>/vendor/ezsystems/ezpublish-kernel/data/update/mysql/dbupdate-6.13.0-to-7.5.7.sql`
```

> But you should for instance also create new section for "Forms" (2.4) etc. and everything else explained there in order to get a fully functional 2.5.

Yes I'm missing that one, as mentioned above, there is also a few other things we should cover.

Otherwise I think I cover everything else relevant when coming from 5.4, so to go true all of it then:
- KO: `<1.7 Changes to Solr Bundle` => AFAIK invalid, there are more changes to schema then what is mentioned
    - So we should add own section here on using the new solr schema instead
- OK: `<1.13 Changes to permissions` => Covered by dbupdate-5.4.0-to-6.13.0.sql AFAIK
- OK: `<1.13 Changes to form-uploaded files` => Irrelevant, is about LandingPage
- OK: `<1.13 ezsearch_return_count table removal` => This was covered in 5.4, we could add it again if we want
- OK: `<1.13 Increased password hash length` => Should be covered by dbupdate-5.4.0-to-6.13.0.sql AFAIK
- OK: `<2.2 Change from UTF8 to UTF8MB4` => Covered in the links here
- OK: `<2.2 Page builder` => Not relevant as we install the table clean with all fixes in step to install EE tables here
- OK: `<2.2 Migrate Landing Pages` => Covered below here
- OK: `<2.3 Changes to timestamp` => Covered by the NEW (PR) `dbupdate-6.13.0-to-7.5.7`
- KO: `<2.3 Form builder` => We should add that step here or link to it, as described in comment above.
- OK: `<2.4 Workflow` => Covered by instructions to install all EE tables
- ??: `<2.4 Changes to the Forms folder` => Kind of assume command above to create folder uses right section, but worth checking
- OK: `<2.4 Changes to Custom tags` => Irrelevant, install does not yet use RichText


Then there is 2.5 page:
- OK: All DB steps are covered by `dbupdate-6.13.0-to-7.5.7`
- OK: `DoctrineSchemaBundle ` is already enabled as we instruct people to use clean 2.5.x as starting point here
- OK: matrix steps is covered here
- ??: Cache flush can be relevant, even if not for the same reasons, so we should rather add section somewhere during the steps to clear cache here using symfony cache pool command
- OK: 2.5.3 page builder is covered in instructions here
- OK: 2.5.6 Password expiration should be covered by `dbupdate-6.13.0-to-7.5.7`